src/python/WMCore/JobStateMachine/SummaryDB.py

```diff
@@ -71,7 +71,8 @@ def fwjr_parser(doc):
     pdict = dict(totalJobCPU=0, totalJobTime=0, totalEventCPU=0)
     ddict = {}
     for key, val in steps.items():
-        wrappedTotalJobTime += val['stop'] - val['start']
+        if val['stop'] is not None and val['start'] is not None:
```

Comment:
shouldn't this be 'or' not 'and'?

Reply:
I just copied your code from PR 8057 :-) But I think it's and, since we want to make sure both start/stop values are not null, before we subtract them.

Reply:
Sorry, I misread. I thought "is None or is None". My bad.